olm-catalog/serverless-operator/index/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN /bin/opm render --skip-tls-verify -o yaml registry.ci.openshift.org/knative/
 
 # The base image is expected to contain
 # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-FROM quay.io/openshift/origin-operator-registry:4.15
```

Comment:
:smile:  That's quiet a good fix.

Do you want that on 1.32 too, @dsimansk ?

Reply:
yes please ;)

Reply:
Sure, we can do that certainly.

Reply:
Neverming the cherry-pick will probably fail due to main being on 4.15 OCP_MAX variable. I'll defer to manual change then. :)

Reply:
--> https://github.com/openshift-knative/serverless-operator/pull/2545